src/components/toast/_toaster.scss

```diff
@@ -1,25 +1,31 @@
 // --- <b-toaster> custom SCSS ---
 
-// Configured "toasters".
+// Configured "toasters":
 // The following are columnar toasters (vertically stacked)
+// and fixed within the viewport
 //   b-toaster-top-right
 //   b-toaster-top-left
 //   b-toaster-bottom-right
 //   b-toaster-bottom-left
 //
-// Note: This may change to using a flex layout
-//
+// NOTE:
+// This SCSS is prelimiary, and may change in the future.
+// We may add an inner wrapper inside .b-toast (a .b-toaster-slot)
+// To allow for better customization of positional styling.
 
 .b-toaster {
   position: fixed;
-  max-width: $toast-max-width;
+  min-width: $b-toaster-min-width;
+  max-width: $b-toaster-max-width;
   overflow: visible;
   z-index: $b-toaster-zindex;
 
   // Ensure that an empty toaster isn't occupying any space
   // and obscuring access to underlying elements
   &:empty {
     padding: 0 0;
+    margin: 0 0;
+    display: none;
```

Comment:
Why reset the `padding` and `margin` when we set it to `display: none;` anyway?

Reply:
Just in case users have defined their own toasters definition.

Reply:
Was also going to add in `width: 0 ; height: 0;`

Reply:
The margin and padding removal is to make sure the toaster container is moved to the very edge of the screen.